### ISSUE TITLE
Analytics to track how much time graphic spends in view

### ISSUE THREAD

**Issue by mitchthorson** [USATODAY/usat-graphics-kit#7]
Create a script on parent pages that will send messages to embed when graphic enters and exits the window. Use these messages to start and stop a timer that sends analytics events at regular intervals... 10s, 15s, 30s. This will allow us to see how long people spend looking at embedded graphics in a story.

**Comment by pimlinders** [comment]
@mitchthorson done?

**Comment by mitchthorson** [comment]
@pimlinders  not quite... the idea with this one is to track when an embedded graphic (on a story page) is within the viewport only, not just total time on page.
